### ISSUE TITLE
<li> not in <ul> element - HTML validation error

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#770]
Axe audit finds this in the masthead:

```
<span class="navbar-nav" aria-label="browse" role="navigation">
   <li class="nav-item ">
      <a class="nav-link" href="/repositories">Repositories</a>
   </li>
   <li class="nav-item ml-3 ">
      <a class="nav-link" href="/collections">Collections</a>
   </li>
</span>
```

The list items need a `<ul>` parent. Switching `<span>` to `<ul>` doesn't seem to break anything.
